src/node/rpc/frontend.h

```diff
@@ -718,14 +718,19 @@ namespace ccf
       return jsonrpc::pack(rep.value(), pack.value());
     }
 
+    std::string get_method(const std::string& method)
+    {
+      return method.substr(method.find_last_of('/') + 1, method.size());
```

Comment:
If `method` doesn't contain `/`, this returns the entire string (`find_last_of` returns npos aka max(size_t), +1 wraps to 0) - is this the intended behaviour? I think `RpcEndpoint::get_actor` should return the actor _and_ the remaining method, rather than repeating a similar parse here.

Reply:
That's the idea once this disappears, the method becomes an input to process. This is only here to get stuff working with the existing code path/validate the idea.